src/server/scripts/Northrend/zone_sholazar_basin.cpp

```diff
@@ -663,6 +663,125 @@ class spell_q12611_deathbolt : public SpellScript
     }
 };
 
+/*######
+## Quest 12683: Burning to Help
+######*/
+
+enum BurningToHelp
+{
+    SPELL_HYDRA_SPUTUM     = 52307
+};
+
+// 52308 - Take Sputum Sample
+class spell_sholazar_take_sputum_sample : public SpellScript
+{
+    PrepareSpellScript(spell_sholazar_take_sputum_sample);
+
+    bool Validate(SpellInfo const* spellInfo) override
+    {
+        return ValidateSpellInfo(
+        {
+            uint32(spellInfo->GetEffect(EFFECT_0).CalcValue()),
+            uint32(spellInfo->GetEffect(EFFECT_1).CalcValue())
+        });
+    }
+
+    SpellCastResult CheckCast()
+    {
+        if (!GetCaster()->HasAura(uint32(GetEffectInfo(EFFECT_1).CalcValue())))
+            return SPELL_FAILED_CANT_DO_THAT_RIGHT_NOW;    // Should be 'You can't do that yet', can't find it
```

Comment:
SPELL_FAILED_CASTER_AURASTATE = "You can't do that yet";
SPELL_FAILED_TARGET_AURASTATE = "You can't do that yet";

Reply:
Nice
Thanks